### ISSUE TITLE
Overlay: Centred popup (Lightbox) content from previous page not hidden for access technology

### ISSUE THREAD

**Issue by dfarough** [wet-boew/wet-boew#7821]
@wet-boew/access-working-group
when running this example the content from the previous page is still exposed to users of access technology.
This will be confusing especially if the content in the dialogue does not have the focus.  Can we insure that the content from the underlying page is hidden when the overlay is displayed?
